fe/fe-core/src/main/java/org/apache/doris/analysis/CreateJobStmt.java

```diff
@@ -81,16 +81,17 @@ public class CreateJobStmt extends DdlStmt {
     private final String endsTimeStamp;
 
     private final String comment;
+    private JobExecuteType executeType;
 
     private String timezone = TimeUtils.DEFAULT_TIME_ZONE;
 
     private static final ImmutableSet<Class<? extends DdlStmt>> supportStmtSuperClass
             = new ImmutableSet.Builder<Class<? extends DdlStmt>>().add(InsertStmt.class)
```

Comment:
Use InsertIntoCommand

Reply:
This will be implemented together in insertjob